index.html

```diff
@@ -189,7 +190,7 @@ <h2>About Me</h2>
 				</div>
 			</div>
 
-			<div class="col-md-1"></div>
+			<div className="col-md-1"></div>

```

Comment:
In React, the attribute should be className instead of class. Update to <div className="col-md-1"></div>.